packages/broker/src/broker.ts

```diff
@@ -40,7 +54,7 @@ export const startBroker = async (config: Config): Promise<Broker> => {
     if (!wallet) {
         throw new Error('Could not resolve Ethereum address from given config.ethereumPrivateKey')
     }
-    const brokerAddress = wallet.address
+    const brokerAddress = `${wallet.address}#${uuidv4()}`
```

Comment:
We should probably rename this to something like `brokerId` `brokerUid` or `brokerUUID` or something